web/utilities.py

```diff
@@ -172,11 +172,21 @@ def pull_picture(id):
     return image
 
 def sendConfirmationEmailToUser(entry):
-    msg = Message(f"ArtBot Agar Art Submission on Behalf of {entry.email}")
+    msg = Message(f"ARTBot Submission Confirmation for {entry.email}")
```

Comment:
Do we need to have entry.email (users email) in the subject line?

Reply:
No, I don't think so. Maybe we use entry.title instead?